.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.5-dev"  # 3.5 development branch
+  - "3.6"
+  - "3.6-dev"  # 3.6 development branch
+  - "3.7-dev"  # 3.7 development branch
+# command to install dependencies
+install:
+  - pip install -r Requirements.txt
+# command to run tests
+script:
```

Comment:
either use nosetests, or pytest to run your test in this scripts section but make sure you include either of them in the requirment.txt file